app/Infrastructure/Http/Controllers/DashboardController.php

```diff
@@ -85,9 +88,11 @@ public function __invoke(
         return inertia("Dashboard", [
             "absences" => VacationRequestResource::collection($absences),
             "remoteDays" => VacationRequestResource::collection($remoteDays),
+            "birthdays" => BirthdayResource::collection($birthdays),
             "upcomingAbsences" => VacationRequestResource::collection($upcomingAbsences),
             "upcomingRemoteDays" => VacationRequestResource::collection($upcomingRemoteDays),
             "vacationRequests" => VacationRequestResource::collection($vacationRequests),
+            "upcomingBirthdays" => BirthdayResource::collection($upcomingBirthdays),
```

Comment:
There's a lot of xs and upcoming xs here. Maybe some aggregator with methods `getCurrent()` and `getUpcoming()` would be useful?